### PR TITLE
#2734 - Ecert File Revisions

### PR DESCRIPTION
**Pull request by guru-aot**
- [x] Confirm / Update eCert creation for PT & FT to include PD/PPD flag (potentially PT was already done)

PT eCert PD/PPD flag populating based on application selection
FT eCert PD/PPD flag populating based on application selection

### CODE REVIEW

**Review by dheepak-aot — APPROVED** [review]
Looks good 👍

**Review by lewischen-aot — APPROVED** [review]
LGTM👍

File: sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/_tests_/unit/e-cert-full-time-integration.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("Should create content for full time E-CERT file when requested providing E-CERT records.", async () => {
+    // Arrange
+    const ecertRecords = [
+      {
+        sin: "445966120",
+        stopFullTimeBCFunding: false,
```
Comment (sh16011993) [inline sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/_tests_/unit/e-cert-full-time-integration.service.spec.ts:42]: @guru-aot Is this property needed? I don't it as a part of the interface `ECertRecord`.

**Review by sh16011993 — COMMENTED** [review]
Nice work @guru-aot 👍 Added a minor question comment.

File: sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/_tests_/unit/e-cert-full-time-integration.service.spec.ts (hunk 1 above)
Comment (guru-aot) [reply sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/_tests_/unit/e-cert-full-time-integration.service.spec.ts:42]: you are right @sh16011993, i will remove it.

**Review by andrepestana-aot — APPROVED** [review]
LGTM!

**Review by sh16011993 — APPROVED** [review]
LGTM @guru-aot 👍

File: sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/_tests_/unit/e-cert-full-time-integration.service.spec.ts (hunk 1 above)
Comment (guru-aot) [reply sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/_tests_/unit/e-cert-full-time-integration.service.spec.ts]: you are right @sh16011993

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Pull request merged**